src/x86/sse.rs

```diff
@@ -1,5 +1,7 @@
 use v128::*;
 
+use simd_llvm::simd_shuffle4;
```

Comment:
Please group this with the other crate imports in alphabetical order. :-) So it should be:

```rust
use simd_llvm::simd_shuffle4;
use v128::*;
```

Thanks!

Reply:
Will do!